### DISCUSSION TITLE
nextpnr-ecp5 not in path

### DISCUSSION THREAD

**Discussion by tarik-hamedovic** [amaranth-lang/amaranth#977] [Q&A] [answered]
Hello 
When i try to run the `python3 -m amaranth_boards.ulx3s 12F` command i get the following error :

> Could not find required tool nextpnr-ecp5 in PATH. Place it directly in PATH or specify path explicitly via the NEXTPNR_ECP5 environment variable.

 I installed prjtrellis and nexpnr. Am I missing something?

**Comment by whitequark** [comment]
What happens if you run `nextpnr-ecp5 --version`?

**Reply by tarik-hamedovic** [reply]
`nextpnr-ecp5: command not found`
I followed this install here: https://libre-soc.org/HDL_workflow/nextpnr/
Just that I used these commands from the official github nextpnr:
```
cmake . -DARCH=ecp5 -DTRELLIS_INSTALL_PREFIX=/usr/local
make -j$(nproc)
sudo make install
```

**Reply by whitequark** [reply]
Try `export PATH=/usr/local/bin:$PATH` and re-run the command.

**Reply by tarik-hamedovic** [reply]
Still giving `nextpnr-ecp5: command not found`

**Reply by whitequark** [reply, marked as answer]
You should look at the output of `sudo make install` and find out where `nextpnr-ecp5` gets installed then.
